### ISSUE TITLE
toolbarSticky not working

### ISSUE THREAD

**Issue by reganlawton** [froala/Craft-3-Froala-WYSIWYG#14]
It seems that the sticky toolbar doesn't work. This is quite annoying for large content snippets. Do we have a fix of work around for it?

**Comment by bertoost** [comment]
I understand. I have to figure that out. Hopefully I can get to it tomorrow. Hang on!

**Comment by reganlawton** [comment]
Thanks for looking into this one. I’ll let the client know this is being looked into. Again thanks for your support.

**Comment by bertoost** [comment]
Hm I can't figure it out either.. Seems not to work at all, even with a `scrollableContainer` set to it's parent container element with ID `#content`.
I will contact Froala team about it.

**Comment by reganlawton** [comment]
Hmm yeah it’s a weird one 🤷‍♂️

**Comment by dejanmartinovic** [comment]
Editor itself cannot be set as the scrolalbleContainer. 
You could put there a parent which is scrollable.
Please find a simple example here: https://jsfiddle.net/dejan_m/Lzq1b3jn/4/
Best, Dejan

**Comment by bertoost** [comment]
Thanks @dejanmartinovic ! I understand. But I tried with it's parent element as scrollable container but that didn't work either

**Comment by reganlawton** [comment]
I’m sick today but I might give this a looksie tomorrow if I’m feeling better. I did see that it does initialises the sticky toolbar just doesn’t trigger it might need some additional code.

Also IDs won’t work as my case we use it inside a Matrix field and which need to support multiple text blocks. This also has another issue where it could have multiple of toolbars layered But this said if the toolbar stickiness is only till the bottom of the container this could be fine.

**Comment by bertoost** [comment]
@reganlawton I am testing it with the #content ID of Craft. It's the container for all fields. But that's not working either for a normal field (not matrix).

Ps. Hopefully you get better soon

**Comment by reganlawton** [comment]
Yeah no have slept through the worst of it. I’ll have a look at it tomorrow I might go into the office early and see what I can do.

**Comment by bertoost** [comment]
FYI; I can't get it to work. I guess it has something to do with how Craft's CP is working (css's flex-box) regarding how Froala's editor is checking the scrolling position. I can imagine this is annoying, but I am getting a little bit out of options.

**Comment by reganlawton** [comment]
@bertoost I haven't gotten a chance to troubleshoot with the new information, been smashed with projects lately. Let's hope I get more luck I'm on the project this is affecting later this week and I'll assign some out of office hours towards it.
